README.md

```diff
@@ -108,3 +109,18 @@ Move into the repo and install the contract dependencies
 Tenderly is used to simulate transactions.
 If you don’t already have a Tenderly account, go to https://dashboard.tenderly.co/login and sign up.
 The free account is sufficient.
+
+## Creating a Task
+
+Each task in the `tasks` directory should contain all of the information required to both validate
+and perform the network interaction. To validate that the intention of the network interaction is
+correct, specific contextual information must be included in `Validation.md` that enables signers
+to double check the correctness. This includes linking to authoritative sources such as the
+[superchain-registry](https://github.com/ethereum-optimism/superchain-registry) or
+[Etherscan](https://etherscan.io) to prove the correctness of particular configuration values.
```

Comment:
Adding the monorepo, as it is referenced by tag in governance proposals:

```suggestion
to double check the correctness. This includes linking to authoritative sources such as the
[superchain-registry](https://github.com/ethereum-optimism/superchain-registry), the [Optimism monorepo](https://github.com/ethereum-optimism/optimism), and
[Etherscan](https://etherscan.io) to prove the correctness of particular configuration values.
```

Reply:
We should be migrating away from using the monorepo as a source of truth for anything except bytecode validation, so I'd suggest linking to the governance forum here instead. And even for bytecode validation, the starting point would be a gov forum post, which then links to a release in the monorepo

Reply:
The place I think we'll have the most difficulty avoiding the monorepo is for storage layout references which we do in various locations for some runbooks.

Reply:
Yea I think storage layouts/snapshots are the only time we should need to reference it

Reply:
OK. Please let me know how you think I should resolve this thread, and anything else for the PR itself.